src/components/settings/SettingsAccessTab/index.jsx

```diff
@@ -23,6 +26,13 @@ const SettingsAccessTab = ({
     updateSubsidyRequestConfiguration,
   } = useContext(SubsidyRequestConfigurationContext);
 
+  const {
+    couponsData,
+    customerAgreement,
```

Comment:
I think you want `customerAgreement: { subscriptions }` here, and then use `subscriptions={subscriptions}` on line  50 below.

Reply:
Fixed